fipy/terms/term.py

```diff
@@ -210,6 +210,11 @@ def sweep(self, var=None, solver=None, boundaryConditions=(), dt=None, underRela
             :math:`\mathsf{L}\vec{e}=\vec{r}` for the error vector
             :math:`\vec{e}` and store it in the `errorVector` member of
             `Term`
+        
+        Returns
+        -------
+        residual : ~fipy.variables.cellVariable.CellVariable
+            The residual vector :math:`\vec{r}=\mathsf{L}\vec{x} - \vec{b}`
```

Comment:
Interesting -- my local build (HTML & LaTeX) renders this (and several other, but not all) instance of `~fipy.variables.cellVariable.CellVariable` as plain text rather than a link to CellVariable.

Reply:
(The math renders correctly.)